shell/lint.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Are you interested to integrate pre-commit hooks directly as I've proposed in https://github.com/tensorflow/tensorflow/pull/48371

Reply:
yeah, do I need to remove the shebang?  I noticed it was absent in the other repos, I assume this is for good reason?

Reply:
Possibly once we hit a certain level of complexity.   We're hoping to standardize across keras-cv, keras-nlp, and keras-tuner. 

@fchollet @qlzh727 thoughts?

Reply:
If you check my PR relies on https://github.com/pre-commit/pre-commit that It could help a bit also to maintain repos in sync on the same linting and also with the local developer setup.

Reply:
P.s. generally this could reduce triagers  roundtrip request for linting on CI fail.

Reply:
Let's open a Github issue for this!  I think this is a good idea but at the moment I lack the cycles to do a full proper CI implementation.  Let's use the minimal version for now and revisit in the near future when we have some real code to test on etc

Reply:
I can have some cycles.. but please open a new ticket on your side with a contribution welcome label so I know that you have evaluated the approach  before I prepare a PR

Thanks

Reply:
If you're able to open a PR, please do -- that sounds super helpful 👍

Reply:
Ok when this is in master I will try to prepare a PR.